concourse/scripts/publish.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+set -eou
+
+cd ./fauna-jvm-repository
+
+PACKAGE_VERSION=$(sbt faunadb-common/version | tail -n 1 | cut -c 8-)
+
+echo "Going to publish version: $PACKAGE_VERSION"
+
+sbt faunadb-common/publishSigned
+sbt faunadb-java/publishSigned
+sbt faunadb-scala/publishSigned
```

Comment:
```suggestion
sbt +faunadb-scala/publishSigned
```

this will ensure we are releasing with all the Scala versions

Reply:
https://github.com/fauna/faunadb-jvm/blob/master/project/Settings.scala#L13

Reply:
oh, thanks a lot @evbruno 
you've just saved me :)